translate/storage/factory.py

```diff
@@ -61,6 +61,8 @@
 _ext is a pseudo extension, that is their is no real extension by that name.
 """
 
+_classes_str = classes_str
```

Comment:
Wouldn't be better to directly define the dict as _classes_str above instead of defining it as classes_str and then assigning to _classes_str.

That would need some other updates in the code, but is IMHO cleaner approach. Anyway translate/convert/pot2po.py should use None after this change instead of passing factory.classes_str as default.

Reply:
The reason for the double-shuffle assignment is that removing `classes_str` and `hiddenclasses` from factory.py is a non-backwards compatible change. Can we be sure that nothing is using it? (ttkit, pootle, virtaal and lokalize all seem to be OK; if that's enough checking, then let's do that.)